modules/ROOT/pages/waf-policy.adoc

```diff
@@ -0,0 +1,8 @@
+= Web Application Firewall Security Policy
+
+The Web Application Firewall (WAF) security policy is available for Runtime Fabric inbound traffic to provide protection at the Web application level. WAF policies provide the xref:https://www.owasp.org/index.php/Category:OWASP_ModSecurity_Core_Rule_Set_Project[OWASP Core Rule Set (CRS)] functionality checking for common web application attacks in request and response traffic, which provides an industry standard security baseline. Policies are grouped into the major threat categories for requests and responses.
```

Comment:
May want to change "inbound traffic" to "traffic". The term "inbound" might be confusing since both requests and responses can be scanned by WAF.